<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_ImagePropertiesEditingWidget</class>
 <widget class="QWidget" name="LC_ImagePropertiesEditingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="buttonGroup8">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Geometry</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,5,0">
      <item row="0" column="0">
       <widget class="QLabel" name="lInsertX">
        <property name="text">
         <string>Insert Point(x):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leInsertX">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>X coordinate of insertion point</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="LC_PointPickButton" name="wPickInsertionPoint" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>46</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Pick insertion point coordinates from drawing</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lInsertY">
        <property name="text">
         <string>Insert Point (y):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leInsertY">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Y coordinate of insertion point</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lWidth">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leWidth">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Width of image</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="tbPickWidth">
        <property name="toolTip">
         <string>Pick width value from drawing</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/icons/icons.qrc">
          <normaloff>:/icons/interactive_pick_distance.lci</normaloff>:/icons/interactive_pick_distance.lci</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lHeight">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leHeight">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Height of image</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="tbPickHeight">
        <property name="toolTip">
         <string>Pick height value from the drawing</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/icons/icons.qrc">
          <normaloff>:/icons/interactive_pick_distance.lci</normaloff>:/icons/interactive_pick_distance.lci</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lScale">
        <property name="text">
         <string>Scale:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="leScale">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Scaling factor</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lAngle">
        <property name="text">
         <string>Angle:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="leAngle">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Rotation angle</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QToolButton" name="tbPickAngle">
        <property name="toolTip">
         <string>Pick angle from the drawing</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/icons/icons.qrc">
          <normaloff>:/icons/interactive_pick_angle.lci</normaloff>:/icons/interactive_pick_angle.lci</iconset>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lPath">
        <property name="text">
         <string>File Path:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lePath">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Path to image file</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="pbFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select an image file</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/icons/icons.qrc">
          <normaloff>:/icons/fileopen.lci</normaloff>:/icons/fileopen.lci</iconset>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lSize">
        <property name="text">
         <string>Size (px):</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="leSize">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Size of image in pixels</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lDPI">
        <property name="text">
         <string>DPI</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="leDPI">
        <property name="toolTip">
         <string>Resolution (dots per inch)</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="spacer58">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2">
       <widget class="QToolButton" name="tbPickScale">
        <property name="toolTip">
         <string>Pick scale from drawing</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/icons/icons.qrc">
          <normaloff>:/icons/interactive_pick_distance.lci</normaloff>:/icons/interactive_pick_distance.lci</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LC_PointPickButton</class>
   <extends>QWidget</extends>
   <header>lc_pointpickbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../res/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
